drake/solvers/SNOPTWrapper.cpp

```diff
@@ -0,0 +1,343 @@
+
```

Comment:
can you open an issue with a few specific pointers, so that we can ask for help in trying that?